api/v1beta2/foundationdb_database_configuration.go

```diff
@@ -737,6 +737,8 @@ const (
 	StorageEngineRocksDbV1 StorageEngine = "ssd-rocksdb-v1"
 	// StorageEngineShardedRocksDB defines the storage engine ssd-sharded-rocksdb.
 	StorageEngineShardedRocksDB StorageEngine = "ssd-sharded-rocksdb"
+	// StorageEngineRedwood1Experimental defines the storage engine ssd-redwood-1-experimental.
+	StorageEngineRedwood1Experimental StorageEngine = "ssd-redwood-1-experimental"
```

Comment:
Those changes are not required and the operator already supports the Redwood storage engine. The actual validation is done here: https://github.com/FoundationDB/fdb-kubernetes-operator/blob/main/api/v1beta2/foundationdb_database_configuration.go#L42. If we add this constant we should also add it to the `IsStorageEngineSupported` check: https://github.com/FoundationDB/fdb-kubernetes-operator/blob/main/api/v1beta2/foundationdb_version.go#L184.